server/models/record.model.ts

```diff
@@ -0,0 +1,45 @@
+import mongoose, { Schema, Document, Model } from 'mongoose';
+import {IChunkScheme} from "./chuck.model";
+import { v4 as uuid } from 'uuid';
+
+export interface IRecord extends Document {
+    recordId: string;
+    userId: string;
+    startTime?: Date;
+    endTime?: Date;
+    summary?: string;
+    chunks?: IChunkScheme[];
+}
+
+const RecordSchema = new Schema<IRecord>(
+    {
+        recordId: {
+            type: String,
+            ref: 'User',
```

Comment:
You mistakenly referred to the `User` in the `recordId`, you should have `userId`. `id` will be generated by default for the record